### PR TITLE
add generator for nsm site notes & generate notes

### PR DESCRIPTION
**Pull request by denis-tingaikin**
Closes https://github.com/networkservicemesh/site/issues/251

### CODE REVIEW

File: content/docs/releases/v1.11.0.md [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+###  How can i make composition of endpoints?
+
+Status: RESOLVED.
+
+[See more details](https://github.com/networkservicemesh/deployments-k8s/issues/9676)
+
+
+### IPSec takes a lot of time on request
+
+Our current implementation uses IKEv2, that requires `rsa` key.
```
Comment (LionelJouin) [inline content/docs/releases/v1.11.0.md:84]:
```suggestion
Our current implementation uses IKEv2 which requires an `rsa` key. 
```

File: content/docs/releases/v1.11.0.md [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+
+Scenarios:
+- Forwarders death in floating interdomain scenario
+- NSE death in floating interdomain scenario
+- NSM systems death in floating interdomain scenario
+- Proxy nsmgrs death in interdomain scenario
+- NSMGRs death in interdomain scenario
+- Registry death in interdomain scenario
+
+We need to check scenarious from single cluster testing in multicluster scenarious.
```
Comment (LionelJouin) [inline content/docs/releases/v1.11.0.md:42]:
```suggestion
We need to check scenarios from single cluster testing in multi cluster scenarios.
```

File: content/docs/releases/v1.11.0.md [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+- [Bare metal in Equinix Metal](https://github.com/networkservicemesh/integration-k8s-packet/actions?query=branch%3Arelease%2Fv1.11.0+)
+- [Interdomain GKE/AWS/AKS](https://github.com/networkservicemesh/integration-interdomain-k8s/actions?query=branch%3Arelease%2Fv1.11.0+)
+
+## Changes since last release
+
+### Add LoadBalancer for vl3 networks
+
+NSM vl3 could be perceived as an alternative to k8s networks. Currently, k8s service is the most useful thing in the k8s network.
+
+At this moment, NSM vl3 netowk doesn't provide any alternative to k8s services. So we need to consider and implement the best solution for it.
```
Comment (LionelJouin) [inline content/docs/releases/v1.11.0.md:26]:
```suggestion
At this moment, NSM vl3 network doesn't provide any alternative to k8s services. So we need to consider and implement the best solution for it.
```

File: content/docs/releases/v1.11.0.md (hunk 1 above)
Comment (denis-tingaikin) [reply content/docs/releases/v1.11.0.md]: Fixed, thanks.

File: content/docs/releases/v1.11.0.md (hunk 3 above)
Comment (denis-tingaikin) [reply content/docs/releases/v1.11.0.md]: Fixed, thanks.

File: content/docs/releases/v1.11.0.md (hunk 2 above)
Comment (denis-tingaikin) [reply content/docs/releases/v1.11.0.md]: Fixed, thanks.

**Review by glazychev-art — APPROVED** [review]

**Pull request merged**